component/component.js

```diff
@@ -165,14 +165,17 @@ define('ui/components/machine/driver-%%DRIVERNAME%%/component', ['exports', 'emb
           return;
         }
         var templates = listTemplatesResponse.data.filter(function(template) {
-           return template.name.toLowerCase().indexOf('windows') == -1;
+           return template.name.toLowerCase().indexOf('windows') == -1 && template.name.toLowerCase().indexOf('centos 6') == -1;
```

Comment:
You could pull out the regex to make it a constant outside the loop. It would make it less suboptimal ;)